src/Magnum/Audio/Context.h

```diff
@@ -41,6 +52,37 @@ typedef struct ALCcontext_struct ALCcontext;
 
 namespace Magnum { namespace Audio {
 
+class Context;
+
+/**
+@brief Run-time information about OpenAL extension
+
+Encapsulates runtime information about OpenAL extension, such as name string,
+minimal required OpenAL version and version in which the extension was adopted
+to core.
+
+See also @ref Audio::Extensions namespace, which contain compile-time information
+about OpenAL extensions.
+*/
+class MAGNUM_AUDIO_EXPORT Extension {
+    friend Context;
+
+    public:
+        /** @brief All OpenAL extensions */
+        static const std::vector<Extension>& extensions();
+
+        /** @brief Extension string */
+        constexpr const char* string() const { return _string; }
+
+    private:
+        /* GCC 4.6 doesn't like const members, as std::vector doesn't have
+           proper move semantic yet */
```

Comment:
Huh. This is a leftover comment from a long ago and doesn't make any sense even in `Magnum::Extension` :)